bin/deku_node.re

```diff
@@ -223,7 +223,7 @@ let node = {
       |> List.mapi((i, validator) => {
            (
              switch (validator) {
-             | Tezos_interop.Key.Ed25519(k) => k
+             | k => k
              },
```

Comment:
This switch statement seems like a noop. I guess the tuple could simply be `(validator, sprintf("...", i))`